WalletWasabi/WabiSabi/Client/Banning/CoinPrison.cs

```diff
@@ -0,0 +1,89 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.Helpers;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.WabiSabi.Client.Banning;
+
+public class CoinPrison
+{
+	public CoinPrison() : this("")
```

Comment:
wtf :)

Reply:
Ctor used for testing, but I'll remove this and use `""` as parameter for test cases.